scan/src/App.re

```diff
@@ -96,8 +96,8 @@ let make = () => {
          | AccountIndexPage(address, hashtag) => <AccountIndexPage address hashtag />
          | ValidatorIndexPage(address, hashtag) => <ValidatorIndexPage address hashtag />
          | NotFound =>
-           ReasonReactRouter.replace("/");
-           React.null;
+           ReasonReactRouter.replace("/notfound");
```

Comment:
Need to change user url to /notfound (Sometimes they mistype some word and don't want to re-type all of them?)